packages/neutrino-middleware-env/index.js

```diff
@@ -0,0 +1,5 @@
+const { EnvironmentPlugin } = require('webpack');
+
+module.exports = ({ config }, envs = []) => config
```

Comment:
Might be cool to unify middleware/presets in some way like that, not a big deal though